src/base.js

```diff
@@ -13,19 +13,27 @@ module.exports = {
     'plugin:import/recommended',
     'plugin:promise/recommended',
     'plugin:unicorn/recommended',
-    'prettier',
     'plugin:prettier/recommended',
+    'prettier',
   ],
   rules: {
-    // Define custom parcelLab rules
+    'no-param-reassign': ['error', { props: false }],
```

Comment:
Though your explanation does not convince me because reassigning parameters can be very confusing and cause a lot of problems, I am ok disabling it for the `props` as this is something that might happen in React codebases I assume, thus it might make sense in that context.

But I do not want to have that approach in non-react codebases, so please, add it in the `react.js` file.

Reply:
Ok, moved to `react.js` ✅

> reassigning parameters can be very confusing and cause a lot of problems
I think that giving a new name to a reference of an object is more confusing than changing it directly.
```js
const person = { name: 'Jack', surname: 'Bauer' };
const newPerson = person;
newPerson.name = 'Chloe';
console.log(person); // {name: 'Chloe', surname: 'Bauer'}
console.log(newPerson); // {name: 'Chloe', surname: 'Bauer'}
```